examples/all/lcd_test.cc

```diff
@@ -5,18 +5,58 @@
 #include <thread.h>
 
 #include "../../libraries/lcd.hh"
+#include "../snake/cherry_bitmap.h"
 #include "lowrisc_logo.h"
 
+using namespace sonata::lcd;
+
+// Positions to draw messages on the screen
+static constexpr Point TopMessagePos       = {12, 8};
+static constexpr Point BottomMessagePos    = {38, 114};
+static constexpr Size  BottomMessageOffset = {77, 0};
+
 /// Thread entry point.
 void __cheri_compartment("lcd_test") lcd_test()
 {
-	using namespace sonata::lcd;
+	// Initialise the LCD
+	auto lcd    = SonataLcd();
+	auto screen = Rect::from_point_and_size(Point::ORIGIN, lcd.resolution());
 
-	auto lcd      = SonataLcd();
-	auto screen   = Rect::from_point_and_size(Point::ORIGIN, lcd.resolution());
+	// Draw the lowRISC logo to the LCD
 	auto logoRect = screen.centered_subrect({105, 80});
 	lcd.draw_image_rgb565(logoRect, lowriscLogo105x80);
-	lcd.draw_str({1, 1}, "Hello world!", Color::White, Color::Black);
+
+	// Make a version of the cherry bitmap with a white background.
+	uint8_t __attribute__((aligned(4))) cherryImage10x10WhiteBg[200];
```

Comment:
Do we need an align here? If so, can you use a C++11 attribute specifier?

Reply:
No, was just mimicking the existing code. Upon asking around, it seems that this is not needed. Removing now.